test/new-e2e/tests/agent-metric-logs/log-agent/utils/file_tailing_utils.go

```diff
@@ -123,21 +121,21 @@ func CheckLogs(ls LogsTestSuite, service, content string, expectLogs bool) {
 }
 
 // CleanUp cleans up any existing log files (only useful when running dev mode/local runs).
-func CleanUp(ls LogsTestSuite) {
+func CleanUp(ls LogsTestSuite, logPath string) {
 	t := ls.T()
 	t.Helper()
 	var checkCmd string
 
 	if ls.IsDevMode() {
 		switch ls.Env().RemoteHost.OSFamily {
 		default: // default is linux
-			ls.Env().RemoteHost.MustExecute("sudo rm -f /var/log/hello-world.log")
-			ls.Env().RemoteHost.MustExecute("sudo rm -f /var/log/hello-world.log.old")
-			checkCmd = "ls /var/log/hello-world.log /var/log/hello-world.log.old 2>/dev/null || echo 'Files do not exist'"
+			ls.Env().RemoteHost.MustExecute(fmt.Sprintf("sudo rm -f %s", logPath))
```

Comment:
instead of having two remove commands here, would it make sense to enforce a test directory that we can remove in one go? Like: `/var/log/e2e_test_logs` so we can just run 
`ls.Env().RemoteHost.MustExecute(fmt.Sprintf("sudo rm -f /var/log/e2e_test_logs"))` regardless of what is inside it?

Reply:
Yea that sounds like a good idea :) I'll update.  cc @DDuongNguyen for visibility

Reply:
made some changes, Also requesting an [optional] review from Yoon to make sure I didnt miss anything, since he wrote the original code.